feature-libs/user/account/components/myaccount-v2-user/myaccount-v2-user.component.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component } from '@angular/core';
+import { LoginComponent } from '../login';
+
+@Component({
+  selector: 'cx-myaccount-v2-user',
```

Comment:
[MEDIUM]
For consistency reasons, let's keep "my account" as 2 separate words, which comes down to `my-account` (instead of `myaccount`) when using using kebab-case ( e.g. in selector names, css classes, filenames, etc.).

Reply:
hi @Platonn  - as of now, we have kept 'myaccount' as one work across both the PRs... should we change it to 'my-account' ? both in selectors & in file names?

Reply:
For consistency reasons, yes please.

Reply:
done